Sky Atlas/Sky Atlas.html

```diff
@@ -1132,6 +1419,20 @@ <h1>Sections & Primary Docs</h1>
             MKR holders retain control of their tokens.
           </td>
         </tr>
+        <tr>
+          <td>
+            <dfn>A.0.1
+              - Atlas Preamble - Definitions - yUSDS</dfn>
+          </td>
+          <td>Atlas Preamble - Definitions - yUSDS</td>
+          <td>Core</td>
+          <td>
+            The token used to provide the capital for SKY Staking leverage is called yUSDS. It ensures that all
+            SKY-backed borrowing is funded by segregated risk capital. Users can deposit USDS into a smart contract
+            and receive yUSDS, which can later be converted back to USDS as long as there is unutilized USDS in the
+            yUSDS converter smart contract.
```

Comment:
Don't add line breaks when adding new documents

```suggestion
            The token used to provide the capital for SKY Staking leverage is called yUSDS. It ensures that all SKY-backed borrowing is funded by segregated risk capital. Users can deposit USDS into a smart contract and receive yUSDS, which can later be converted back to USDS as long as there is unutilized USDS in the yUSDS converter smart contract.
```